manage.py

```diff
@@ -367,6 +370,93 @@ def index_advisory_opinions():
                 print("%d docs loaded" % docs_loaded)
         print("%d docs loaded" % docs_loaded)
 
+def get_xml_tree_from_url(url):
+    r = requests.get(url, stream=True)
+
+    with open('temp.zip', 'wb') as f:
+        for chunk in r:
+            f.write(chunk)
+    zip_file = ZipFile('temp.zip')
+
+    with zip_file.open(zip_file.namelist()[0]) as title,\
+     open('title.xml', 'wb') as title_xml:
+        title_xml.write(title.read())
+
+    return ET.parse('title.xml')
+
+def get_title_52_statutes():
+    es = utils.get_elasticsearch_connection()
+
+    title_parsed = get_xml_tree_from_url('http://uscode.house.gov/download/' +
+                    'releasepoints/us/pl/114/219/xml_usc52@114-219.zip')
+    tag_name = '{{http://xml.house.gov/schemas/uslm/1.0}}{0}'
+    for subtitle in title_parsed.iter(tag_name.format('subtitle')):
+        if subtitle.attrib['identifier'] == '/us/usc/t52/stIII':
+            for subchapter in subtitle.iter(tag_name.format('subchapter')):
+                match = re.match("/us/usc/t52/stIII/ch([0-9]+)/sch([IVX]+)",
+                                    subchapter.attrib['identifier'])
+                chapter = match.group(1)
+                subchapter_no = match.group(2)
+                for section in subchapter.iter(tag_name.format('section')):
+                    text = ''
+                    for child in section.iter():
+                        if child.text:
+                            text += ' %s ' % child.text
+                    heading = section.find(tag_name.format('heading')).text.strip()
+                    section_no = re.match('/us/usc/t52/s([0-9]+)',
+                             section.attrib['identifier']).group(1)
+                    pdf_url = 'https://www.gpo.gov/fdsys/pkg/USCODE-2014-' +\
+                              'title52/pdf/USCODE-2014-title52-subtitleIII-' +\
+                              'chap%s-subchap%s-sec%s.pdf'\
+                              % (chapter, subchapter_no, section_no)
+                    doc = {"doc_id": section.attrib['identifier'],
+                           "text": text,
+                           "name": heading,
+                           "no": section_no,
+                           "title": "52",
+                           "chapter": chapter,
+                           "subchapter": subchapter_no,
```

Comment:
I noticed that `subchapter` does not exist for the title 26 statutes, is that by design?